### DISCUSSION TITLE
Drag and drop using the mouse

### DISCUSSION THREAD

**Discussion by 0xAdriaTorralba** [nikitabobko/AeroSpace#755] [questions-and-answers] [answered]
Is there a way to "enable" a drag-and-drop mode using the mouse that ignores the current layout and appends the selected window into the dropped workspace?

For example, if I have a 2 monitor setup, I want to drag and drop using the mouse one window into the other monitor, and that window is appended into the destination workspace (the other monitor) once it's done. 

The current result for this operation is that the window is "attached" to the origin workspace, and once you drop the window into the destination workspace, it goes back to the origin immediatelly.

Is there a workaround or configuration for this?

I was imagining something like `ctrl-alt-d` enables the "drag and drop mode" and you can freely move windows around.

Sorry if there is already a documentation for this, I couldn't find it.

**Comment by hurshnarayan** [comment]
Why is this left unanswered? I'm trynna figure out the same

**Comment by nikitabobko** [comment, marked as answer]
Probably a duplicate of https://github.com/nikitabobko/AeroSpace/issues/634
